static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -571,8 +571,12 @@ $minute-layout-margin-nudge: gs-span(1);
                     width: 100%;
                 }
 
-                .gu-image {
+                .element--inline__outer {
```

Comment:
yeah ok, that works for me i think. it's unorthodox, but it seems to follow the principle?